content/hardware/04.pro/carriers/portenta-mid-carrier/datasheet/datasheet.md

```diff
@@ -0,0 +1,3096 @@
+            <td style="text-align: left;">Mini PCIe connector (J8), High-Density connectors (J1-J2) and breakout header connector<sup>1</sup> (J16)</td>
+        </tr>
+        <tr>
+            <td style="vertical-align: top; text-align: right;">Ethernet (x1)</td>
+            <td style="text-align: left;">RJ45 connector (J18)</td>
+        </tr>
+        <tr>
+            <td style="vertical-align: top; text-align: right;">SPI (x2)</td>
+            <td style="text-align: left;">
+                <ul>
+                    <li>SPI0: Breakout header connector (J15) and High-Density connector (J2)</li>
+                    <li>SPI1: Breakout header connector (J15) and High-Density connector (J2)</li>
+                </ul>
+            </td>
+        </tr>
+        <tr>
+            <td style="vertical-align: top; text-align: right;">I2S (x1)</td>
+            <td style="text-align: left;">Breakout header connector (J14) and High-Density connector (J1)</td>
+        </tr>
+        <tr>
+            <td style="vertical-align: top; text-align: right;">I2C (x3)</td>
+            <td style="text-align: left;">
+                <ul>
+                    <li>I2C0: Breakout header connector (J15) and High-Density connector (J1)</li>
+                    <li>I2C1: Breakout header connector (J15) and High-Density connector (J1)</li>
+                    <li>I2C2: Breakout header connector (J15) and High-Density connector (J2)</li>
+                </ul>
+            </td>
+        </tr>
+        <tr>
+            <td style="vertical-align: top; text-align: right;">CAN bus (x2)</td>
+            <td style="text-align: left;">
+                <ul>
+                    <li>CAN0<sup>2</sup>: Breakout header connector (J14) and High-Density connector (J1)</li>
+                    <li>CAN1<sup>3</sup>: Breakout header connector (J14), High-Density connector (J1), and screw terminal block (J4)</li>
+                </ul>
+            </td>
+        </tr>
+        <tr>
+            <td style="vertical-align: top; text-align: right;">UART (with flow control) (x4)</td>
+            <td style="text-align: left;">
+                <ul>
+                    <li>SERIAL0: Breakout header connector (J14) and High-Density connector (J1)</li>
+                    <li>SERIAL1: Breakout header connector (J14) and High-Density connector (J1)</li>
+                    <li>SERIAL2: Breakout header connector (J14) and High-Density connector (J2)</li>
+                    <li>SERIAL3: Breakout header connector (J14) and High-Density connector (J2)</li>
+                </ul>
+            </td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+<p style="text-align: justify;">
+<sup>1</sup>For debugging purposes only.<br> 
+<sup>2</sup>CAN0 has no CAN PHY; an external one is needed.<br>
+<sup>3</sup>CAN1 has an onboard CAN PHY available through the Carrier's screw terminal block (J4); it can be enabled or disabled via a DIP switch (SW2).<br>
```

Comment:
```suggestion
<sup>1</sup>For debugging purposes only.

<sup>2</sup>CAN0 has no CAN PHY; an external one is needed.

<sup>3</sup>CAN1 has an onboard CAN PHY available through the Carrier's screw terminal block (J4); it can be enabled or disabled via a DIP switch (SW2).
```